ims/einstein/operations.py

```diff
@@ -681,3 +682,26 @@ def run_script(self, img, script):
                                    constants.DEFAULT_SNAPSHOT_NAME)
             self.fs.remove_snapshot(ceph_img_name,
                                     constants.DEFAULT_SNAPSHOT_NAME)
+
+    # TODO: Write tests for this method before merging to main branch
+    @log
+    def get_iscsi_target(self, node_name):
+        try:
+            if not self.is_admin:
+                raise AuthorizationFailedException()
+            provisioned = False
+            provisioned_nodes = self.list_provisioned_nodes()['retval']
+            for node in provisioned_nodes:
+                if node_name == node[0]:
```

Comment:
Checked the function and it returns a list of [image, parent_image]. You can do something like

```
if node_name in [node[0] for node in provisioned_nodes]:
    pass
```
I guess it is cleaner. Unless @ravisantoshgudimetla has something better in mind.

Also it would be better to change the nested list to a list of named tuples. What do you think @ravisantoshgudimetla ?